src/test/java/org/fcrepo/importexport/integration/ExporterIT.java

```diff
@@ -317,6 +321,44 @@ public void testExportCustomPredicatesInbound() throws Exception {
                 createResource(obj1.toString()),
                 createProperty("http://example.org/custom"),
                 createResource(col1.toString())));
+        assertFalse(collectionModel.contains(
+                createResource(other.toString()),
+                createProperty("http://example.org/other"),
+                createResource(col1.toString())));
+    }
+
+    @Test
+    public void testExportInboundFiles() throws Exception {
```

Comment:
@awoods The previous test already demonstrates what you outlined:
* `obj1` is the resource from outside `col1`
* I'm verifying that it is exported at [line 313](https://github.com/fcrepo4-labs/fcrepo-import-export/pull/99/commits/c79a83d135d578ed7ee23c6859a5fe6b8f593493#diff-2085f9c938964a6c935f6979bfc109dbR313)
* I'm verifying that the inbound triple (obj1 ex:custom col1) isn't exported in col1's export on [line 320](https://github.com/fcrepo4-labs/fcrepo-import-export/pull/99/commits/c79a83d135d578ed7ee23c6859a5fe6b8f593493#diff-2085f9c938964a6c935f6979bfc109dbR320)
* And I'm verifying that an inbound triple with a different predicate isn't included in the export on [line 324](https://github.com/fcrepo4-labs/fcrepo-import-export/pull/99/commits/c79a83d135d578ed7ee23c6859a5fe6b8f593493#diff-2085f9c938964a6c935f6979bfc109dbR324)

What I'm adding in c79a83d135d578ed7ee23c6859a5fe6b8f593493 is a test that demonstrates fcrepo-2500 and the fix for it.  I don't think that can be separated from this PR, because d34b5771ff4abeb6ecc52fa1fa528b0117ab1b9f results in all binary descriptions being exported as empty turtle documents (just the namespace definitions survive the filtering).